t/app/controller/reports.t

```diff
@@ -122,6 +123,9 @@ is scalar @$problems, 5, 'correct number of problems displayed';
 FixMyStreet::override_config {
     MAPIT_URL => 'http://mapit.mysociety.org/',
 }, sub {
+    $mech->get_ok('/reports/Electricity_Gas+Council');
```

Comment:
If this was a `follow_link_ok` it'd have the benefit of testing the href generation as well.